.golangci.yml

```diff
@@ -79,12 +79,12 @@ linters-settings:
     allow-multiline-assign: true
     allow-case-traling-whitespace: true
     allow-cuddle-declarations: false
+  godot:
+    check-all: false
```

Comment:
What's this do?

Reply:
The linter will check only `declarations`. e.g license check will be ignored